server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,201 @@
+/* 
+ * Description: Creates ad objects 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+
+import java.lang.StringBuilder;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List; 
+
+public class Ad {
+
+    public String id; 
+    public String advertiser;  
+    public String startDate;
+    public String endDate;
+    public long impressionsMin; 
+    public long impressionsMax; 
+    public boolean ageTargetingEnabled; 
```

Comment:
nit: a better name could be `isTargetingAge`

Reply:
That is a much better name, thank you